src/views/find-room.vue

```diff
@@ -95,25 +101,82 @@ export default {
         this.zoomMap = 5;
         this.markerMap = false;
       }
+      if (results.length == 1) {
+        console.log(results);
+        this.checkRoomExist(results.label);
+      }
     },
     goToAdresse(adresse) {
       this.addresses = [adresse];
       this.center = [adresse.y, adresse.x];
       this.markerMap = true;
       this.zoomMap = 18;
+      this.checkRoomExist(adresse.label);
+    },
+    checkRoomExist(adresse) {
+      console.log(adresse);
+      axios
+        .post("https://nock-nock.herokuapp.com/api/building/filter/adress", {
```

Comment:
J'ai remarquer une faute d'orthographe dans ma requête, du coup j'ai corriger cote back.
C'est donc /filter/address et non /filter/adress

Reply:
Je tombe sur une 404 not found lorsque je tape l'api sur l'url que tu m'as dit

Reply:
Je peux ajouter des user  un building d'ailleurs ?

Reply:
Je viens de deploy ;)
Et oui je t'ai fait ta requete quand tu me l'as demander haha :
{{URL}}api/building/addUser/{{buildingID}}

Body :
{
	"userId" : "5dfc849016ecba19fd9c7fe7"
}